landside/catkin_ws/src/simulation/personal/scenes/robot_code.lua

```diff
@@ -1,5 +1,8 @@
 require "math"
-require "common_code"
```

Comment:
If I'm remembering right, we realized that a lot of code was supposed to have been moved into common_code.lua, but had not been. I believe this branch contains the bad version of this that should not be merged

Reply:
Don't merge